### ISSUE TITLE
[Bug]: 1 duplicate symbol for architecture arm64

### ISSUE THREAD

**Issue by therobertgrigorian** [braze-inc/braze-react-native-sdk#245]
### Which Platforms?

iOS

### Which React Native Version?

0.73.2

### Which @braze/react-native-sdk SDK version?

8.3.0

### Repro Rate

100%

### Steps To Reproduce

Prerequisite:
Install `react-native-health` v1.18.0 and `@braze/react-native-sdk` (version 6 to 8). While compiling, you may encounter the error: "1 duplicate symbol for architecture arm64". We resolved it as follows:

```
diff --git a/node_modules/@braze/react-native-sdk/iOS/BrazeReactBridge/BrazeReactBridge/BrazeReactBridge.mm b/node_modules/@braze/react-native-sdk/iOS/BrazeReactBridge/BrazeReactBridge/BrazeReactBridge.mm
index eab7bc8..0e68270 100644
--- a/node_modules/@braze/react-native-sdk/iOS/BrazeReactBridge/BrazeReactBridge/BrazeReactBridge.mm
+++ b/node_modules/@braze/react-native-sdk/iOS/BrazeReactBridge/BrazeReactBridge/BrazeReactBridge.mm
@@ -32,7 +32,7 @@ @interface BrazeReactBridge () <BrazeSDKAuthDelegate>
 
 @implementation BrazeReactBridge
 
-BOOL hasListeners = NO;
+static BOOL hasListeners = NO;
 
 #pragma mark - Setup
```

### Expected Behavior

It should be compiled.

### Actual Incorrect Behavior

It is not compiled.

**Comment by jerielng** [comment]
Hi @therobertgrigorian, thanks for raising this. We'll look into getting this fixed and will let you know when we have pushed out a fix. Thanks!

**Comment by jerielng** [comment]
Hey @therobertgrigorian, this duplicate symbols error should now be resolved as of version [8.4.0](https://github.com/braze-inc/braze-react-native-sdk/releases/tag/8.4.0). I'll close out this issue, but please let us know if you have any further questions! Thanks!
